Basic MIR tools implementation (#4)
rokugo-lang/rokugo: crates/ir-generated/README.md

# Rokugo IR Generated

> Rokugo's Intermediate Representation, but without suitable, and optimized data structure required by runtime without development kit.

Reviewer: This sentence is written in a pretty convoluted way; I'd suggest:
```suggestion
Rokugo's Intermediate Representation, but without optimized data structures required by the user-facing (non development) runtime.
```
